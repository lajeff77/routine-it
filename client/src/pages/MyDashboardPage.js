import React from "react";
import Dashboard from "../components/Dashboard";
import PermanentSideDrawer from "../components/PermanentSideDrawer";
import {Grid, Item} from '@mui/material';

const MyDashboardPage = () => {

    return (
        <div>
            <Grid container rowSpacing={2} >
                <Grid item xs={2}><PermanentSideDrawer /></Grid>
                <Grid item xs={10}><Dashboard /></Grid>
            </Grid>
        </div>
    )
}

export default MyDashboardPage;