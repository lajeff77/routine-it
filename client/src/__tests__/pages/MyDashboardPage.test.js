import { render} from "@testing-library/react";
import React from "react";
import MyDashboardPage from "../../pages/MyDashboardPage";
import { MemoryRouter, Route, Routes } from "react-router-dom";


describe(MyDashboardPage.name, () => { 
    test('MyDashboardPage matches snapshot', () => {
        const { asFragment } = render(
            <MemoryRouter>
            <Routes>
                <Route element={<MyDashboardPage />} path="/mydashboard" />
            </Routes>
            </MemoryRouter>
        );
    
        expect(asFragment()).toMatchSnapshot();
    });
});