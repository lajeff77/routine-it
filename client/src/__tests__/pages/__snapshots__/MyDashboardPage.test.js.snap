// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyDashboardPage MyDashboardPage matches snapshot 1`] = `<DocumentFragment />`;
